templates/home/home-en.html

```diff
@@ -65,48 +65,28 @@ <h3><a href="https://www.canada.ca/en/services/business.html">Business and indus
-				<p>Food, nutrition, diseases, vaccines, drugs, product safety and recalls</p>
-			</div>
 			<div class="col-lg-4 col-md-6">
 				<h3><a href="https://www.canada.ca/en/services/taxes.html">Taxes</a></h3>
 				<p>Income tax, payroll, GST/HST,  contribution limits, tax credits, charities</p>
 			</div>
 			<div class="col-lg-4 col-md-6">
-				<h3><a href="https://www.canada.ca/en/services/environment.html">Environment and natural resources</a></h3>
-				<p>Weather, climate, agriculture, wildlife, pollution, conservation, fisheries</p>
-			</div>
-			<div class="col-lg-4 col-md-6">
-				<h3><a href="https://www.canada.ca/en/services/defence.html">National security and defence</a></h3>
-				<p>Military, transportation and cyber security, securing the border, counter-terrorism</p>
-			</div>
-			<div class="col-lg-4 col-md-6">
-				<h3><a href="https://www.canada.ca/en/services/culture.html">Culture, history and sport</a></h3>
-				<p>Arts, media, heritage, official languages, national identity and funding</p>
-			</div>
-			<div class="col-lg-4 col-md-6">
-				<h3><a href="https://www.canada.ca/en/services/policing.html">Policing, justice and emergencies</a></h3>
-				<p>Safety, justice system, prepare for emergencies, services for victims of crime</p>
-			</div>
-			<div class="col-lg-4 col-md-6">
-				<h3><a href="https://www.canada.ca/en/services/transport.html">Transport and infrastructure</a></h3>
-				<p>Aviation, marine, road, rail, dangerous goods, infrastructure projects</p>
+				<h3><a href="https://www.canada.ca/en/services/health.html">Health</a></h3>
+				<p>Food, nutrition, diseases, vaccines, drugs, product safety and recalls</p>
 			</div>
 			<div class="col-lg-4 col-md-6">
-				<h3><a href="https://international.gc.ca/world-monde/index.aspx?lang=eng">Canada and the world</a></h3>
-				<p>Foreign policy, trade agreements, development work, global issues</p>
+				<h3><a href="https://www.canada.ca/en/services/environment.html">Environment and natural resources</a></h3>
+				<p>Weather, climate, agriculture, wildlife, pollution, conservation, fisheries</p>
 			</div>
 			<div class="col-lg-4 col-md-6">
 				<h3><a href="https://www.canada.ca/en/services/finance.html">Money and finances</a></h3>
 				<p>Personal finance, credit reports, fraud protection, paying for education</p>
 			</div>
-			<div class="col-lg-4 col-md-6">
-				<h3><a href="https://www.canada.ca/en/services/science.html">Science and innovation</a></h3>
-				<p>Scientific research on health, environment and space, grants and funding</p>
-			</div>
+		</div>
+		<div class="mrgn-top-sm" style="max-width: 140px;">
+			<p style="font-weight: bold; margin-top:20px; margin-bottom:20px; border: 2px solid #26374a; padding: .75em; border-radius: 5px; text-align: center;"><a style="text-decoration: none !important; color:#26374a !important;" href="https://www.canada.ca/en/services.html">All services</a></p>
```

Comment:
No inline style, please add the CSS in the SASS file for the home page.

See there is one SCSS per view port here: https://github.com/wet-boew/GCWeb/tree/master/templates/home

The _base.scss is the default CSS applied for the home page.

You can learn about SASS here: https://sass-lang.com/documentation/syntax/

Reply:
Removed inline css